quiztest/src/main/java/quiztest/main/inventory/DealershipSimulator.java

```diff
@@ -0,0 +1,60 @@
+package quiztest.main.inventory;
+
+import java.sql.Timestamp;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
This is only supposed to be loaded on non-production profiles (I.e. the dev, dev-windows, or debug profile).

Reply:
Good idea. Done.